### PR TITLE
i18n: shortened text strings

### PR DESCRIPTION
**Pull request by math-GH**
Follow up of #5420 that shortened this 2 strings from a sentence to a label

Changes proposed in this pull request:

- i18n

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/zh-tw/admin.php [hunk 1]
```diff
@@ -195,8 +195,8 @@
 		'changelog' => 'Changelog',	// TODO
 		'check' => '檢查更新',
 		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
-		'current_version' => '當前 FreshRSS 版本為 %s。',
-		'last' => '上次檢查：%s',
+		'current_version' => '當前 版本為',
```
Comment (Frenzie) [inline app/i18n/zh-tw/admin.php:198]: Pinging @den13501 for review

Comment (math-GH) [reply app/i18n/zh-tw/admin.php:198]: The line above is more important than this mentioned line ;)

File: app/i18n/zh-tw/admin.php [hunk 2]
```diff
@@ -195,8 +195,8 @@
 		'changelog' => 'Changelog',	// TODO
 		'check' => '檢查更新',
 		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
-		'current_version' => '當前 FreshRSS 版本為 %s。',
```
Comment (math-GH) [inline app/i18n/zh-tw/admin.php:198]: Pinging @den13501 for review

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
Ups, there is a little conflict now. I suggest we merge and come back to it when/if we get feedback.

**Review by den13501 — APPROVED** [review]
Thanks

**Comment by math-GH** [comment]
> Ups, there is a little conflict now. I suggest we merge and come back to it when/if we get feedback.

fixed

**Pull request merged**